kong/globalpatches.lua

```diff
@@ -15,9 +15,11 @@ return function(options)
   local meta = require "kong.meta"
 
 
+  -- TODO: if other modules call require("cjson.safe").new()
```

Comment:
I think we have that in other modules too.

Reply:
yes, and I only found two in kong source code.